docs/modules/ROOT/examples/operator/bundle.yaml

```diff
@@ -63,10 +63,12 @@ spec:
                     - LoadBalancer
                     type: string
                   type:
-                    description: 'Specifies how members are exposed. Valid values are: - "Smart" (default): each member pod is exposed with a separate external address - "Unisocket": all member pods are exposed with one external address'
+                    description: 'Specifies how members are exposed. Valid values are: - "ALL_MEMBERS" (default): each member pod is exposed with a separate external address - "SINGLE_MEMBER": all member pods are exposed with one external address - 
+                    "MULTI_MEMBER': members of a partition group are exposed with a separate external address and act as a gateway to other members in the cluster'
                     enum:
-                    - Smart
-                    - Unisocket
+                    - ALL_MEMBERS
+                    - SINGLE_MEMBER
+                    - MULTI_MEMBER
```

Comment:
@JamesHazelcast is right. We are not updating our side for now. We might update later, but not for 5.5. 

When we will introduce these parameters for next releases, we will just introduce two fo them:
```
  exposeExternally:
    type: Smart/Unisocket --> ALL_MEMBERS/SINGLE_MEMBER
```
Because other option is not applicable for our service topology

Reply:
Confirmed with Hasan - this file can be removed from the PR